### PR TITLE
Make tooltip / hint  on hover example

### PR DESCRIPTION
**Pull request by arekgotfryd**
This pull request adds example of showing tooltip/hint with text information about hovered feature.

### CODE REVIEW

File: examples/tooltip-on-hover.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  source: new VectorSource({
+    url: 'https://openlayers.org/data/vector/ecoregions.json',
+    format: new GeoJSON(),
+  }),
+  background: 'white',
+  style: function (feature) {
+    const color = feature.get('COLOR') || '#eeeeee';
+    style.getFill().setColor(color);
+    return style;
+  },
```
Comment (tschaub) [inline examples/tooltip-on-hover.js:25]:
```suggestion
  style: {
    'fill-color': ['string', ['get', 'COLOR'], '#eeeeee'],
  },
```

We're moving away from using the style and symbolizer constructors.  Can you update the example to use a style like the one above and remove the call to the `Style` constructor above?

**Comment by tschaub** [comment]
Thanks for this nice looking contribution, @arekgotfryd.

File: examples/tooltip-on-hover.js (hunk 1 above)
Comment (arekgotfryd) [reply examples/tooltip-on-hover.js]: Thanks for quick review. Suggested changes applied.

File: examples/tooltip-on-hover.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import Style from '../src/ol/style/Style.js';
+import VectorLayer from '../src/ol/layer/Vector.js';
+import VectorSource from '../src/ol/source/Vector.js';
+import View from '../src/ol/View.js';
+
+const style = new Style({
+  fill: new Fill({
+    color: '#eeeeee',
+  }),
+});
```
Comment (tschaub) [inline examples/tooltip-on-hover.js:13]: This will need to go away too.  And the `import`s for `Fill` and `Style` above.

Comment (arekgotfryd) [reply examples/tooltip-on-hover.js]: Ah sorry for that. Done.

File: examples/tooltip-on-hover.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  } else {
+    info.style.visibility = 'hidden';
+  }
+  currentFeature = feature;
+};
+
+map.on('pointermove', function (evt) {
+  if (evt.dragging) {
+    info.style.display = 'none';
```
Comment (tschaub) [inline examples/tooltip-on-hover.js:53]:
```suggestion
    info.style.visibility = 'hidden';
```

It looks like if you drag the map, the tooltip is not shown again on hover.  I think you'll either want to use `display` or `visibility` consistently.

Comment (arekgotfryd) [reply examples/tooltip-on-hover.js]: Switched to visibility for consistency.

**Review by tschaub — APPROVED** [review]

**Comment by tschaub** [comment]
Thanks for the work on this, @arekgotfryd.  Looks great.

**Pull request merged**